buildconfig/download_msys2_prebuilt.py

```diff
@@ -18,51 +20,66 @@ def install_pacman_package(pkg_name):
     return output.returncode != 0
 
 
-def get_packages(x86=True, x64=True):
+def get_packages(MINGW_ARCH):
```

Comment:
minor nitpick: This should be `mingw_arch` because this is a function argument, and not a constant. This change can also be applied to other functions in this file